.travis.yml

```diff
@@ -1,3 +1,7 @@
 language: ruby
+sudo: false
+cache:
+  bundler: true
+  apt: true
```

Comment:
Hi, I just missed that it was removed (and apparently never worked on the public infrastructure) – and it doesn't sound like it'll be enabled again in the near future.
I'll remove that, no point in keeping it.